README.md

```diff
@@ -6,44 +6,50 @@ Using fluent-plugin-logdna, you can send the logs you collect with Fluentd to Lo
 
 ## Instructions
 
-* Install [Fluentd](http://www.fluentd.org/download)
-* Alternative install if using fluentd package manager (td-agent): `td-agent-gem install fluent-plugin-logdna`
-* Add the contents below to `/etc/fluent/fluent.conf`. For td-agent, use `/etc/td-agent/td-agent.conf`:
-* Alternative install without td-agent is: `gem install fluent-plugin-logdna`
+* Install Fluentd
+  * [Download here](http://www.fluentd.org/download)
+  * If using fluentd package manager (td-agent): `td-agent-gem install fluent-plugin-logdna`
+  * To install without td-agent: `gem install fluent-plugin-logdna`
+* Add the config below to `/etc/fluent/fluent.conf`. For td-agent, `/etc/td-agent/td-agent.conf`:
+
+## Configuration
+
+### Configuration Parameters
+- `api_key`: [Ingestion Key](https://docs.logdna.com/docs/ingestion-key) *Required*
```

Comment:
Just a style q: what is the readme industry standard for describing something as either *Required* or *Optional* — it looks just a bit off imo

Reply:
@alfeng6, I don't think there is such standard

Reply:
Yes, there's a standard.  Things in brackets are optional, and without them is required.  If this were an optional key, it'd be `[api_key]`